storagetransfer/quickstart.py

```diff
@@ -26,8 +26,8 @@
 from google.cloud import storage_transfer
 
 
-def create_one_time_transfer(project_id: str, source_bucket: str,
-                             sink_bucket: str):
+def create_one_time_transfer(project_id: str = "my_project_id", source_bucket: str = "my_source_bucket",
```

Comment:
I personally don't think adding defaults which won't work if user runs this function without parameters is a good practice. Can you elaborate on how this helps with Neos tutorial?

Reply:
Can we instruct the user to uncomment lines 36, 39 and 42 to put their values there? Removing these parameters completely will make it unrunnable from shell and break the convention (we use similar parameters in other samples too).

Reply:
The other languages don't have an uncommenting step so I would have to make a special exception for Python in the Neos tutorial code. It's possible, but not elegant in my opinion. 

I understand that it's against convention of normal samples, but this is a quickstart, which tend to work a little differently, like I showed the Storage Quickstart doesn't have any parameters. 

Having default values that need to be edited for the sample to run is also something we do in other quickstarts, for example here it is in the storage node quickstart: https://github.com/googleapis/nodejs-storage/blob/main/samples/quickstart.js#L17

and here in the python bigtable quickstart: https://github.com/googleapis/python-bigtable/blob/HEAD/samples/quickstart/main.py#L23

Reply:
Thanks for additional context and explanation. I am going to approve this although I still think we should come up with a better way of doing this in the future.